src/test/java/core/basesyntax/PairTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Method;
+
+public class PairTest {
+
+    private Pair firstPair;
+    private Pair secondPair;
+    private Pair thirdPair;
+
+    @Before
+    public void init() {
+        firstPair = Pair.of(5, "Five");
+        secondPair = Pair.of(2, "Two");
+        thirdPair = Pair.of(5, "Five");
+    }
+
+    @Test
+    public void methodsCheck() {
+        Class<Pair> pairClass = Pair.class;
+        Method[] methods = pairClass.getMethods();
```

Comment:
@Bloodstinger не спрацює) getMethods поверне методи і Object також
```suggestion
        Method[] methods = pairClass.getDeclaredMethods();
```
І врахуй що метод of() - це теж метод

Reply:
@Bloodstinger в цьому ж тесті також можеш перевірити що getDeclaredMethods поверне тобі і хешкод і іequals і getFirst....